### PR TITLE
feat: project members

### PR DESCRIPTION
**Pull request by mohammad-alisafaee**

### CODE REVIEW

File: components/renku_data_services/project/api.spec.yaml [hunk 1]
```diff
@@ -271,15 +257,13 @@ components:
         creation_date:
           $ref: "#/components/schemas/CreationDate"
         created_by:
-          $ref: "#/components/schemas/User"
+          $ref: "#/components/schemas/Member"
```
Comment (olevski) [inline components/renku_data_services/project/api.spec.yaml:260]: I don't know when we changed this but the example for this field a few lines below this contains the members property. But the members property is not part of the spec. So we should update that to match.

File: components/renku_data_services/project/api.spec.yaml [hunk 2]
```diff
@@ -223,13 +219,13 @@ paths:
           schema:
             type: string
         - in: path
-          name: user_id
+          name: member_id
```
Comment (olevski) [inline components/renku_data_services/project/api.spec.yaml:222]: Can we be explicit what this is? It is the keycloak ID right?

File: components/renku_data_services/project/orm.py [hunk 3]
```diff
@@ -53,8 +53,28 @@ def dump(self) -> models.Project:
...
             visibility=self.visibility,
-            created_by=models.User(id=self.created_by_id),
+            created_by=models.Member(id=self.created_by_id),
             creation_date=self.creation_date,
             repositories=self.repositories,
             description=self.description,
         )
+
+
+class ProjectMemberORM(BaseORM):
```
Comment (olevski) [inline components/renku_data_services/project/orm.py:63]: What is the purpose of this table? The authz service and database already contains pretty much the same information. 

I was hoping that the authz service and its tables would be the only source of truth for authorization data and membership.

Comment (olevski) [reply components/renku_data_services/project/orm.py:63]: See `components/renku_data_services/authz/orm.py`

**Review by olevski — CHANGES_REQUESTED** [review]
Lets talk about this tomorrow. I think there is a few pieces of code here that just repeat stuff that is already implemented in the authz component.

File: components/renku_data_services/project/orm.py (hunk 3 above)
Comment (mohammad-alisafaee) [reply components/renku_data_services/project/orm.py:63]: As agreed, this will be done in a future PR.

**Review by olevski — CHANGES_REQUESTED** [review]
Can you please fix the tests? Currently only mypy is complaining.

Everything else is ok.

**Review by olevski — APPROVED** [review]

**Pull request merged**